Update zk_login.md to suggest using Base64 instead of BigInt (#14070)
MystenLabs/sui: doc/src/build/zk_login.md

Here's an example request and response for the Mysten Labs-maintained proving service. If you wish to use the Mysten ran ZK Proving Service, please contact us for whitelisting your registered client ID. Only valid JWT token authenticated with whitelisted client IDs are accepted.

> Note that either BigInt or Base64 encoding can be used for the three fields: `extendedEphemeralPublicKey`, `jwtRandomness` and `salt`. The example below shows two sample requests: the first uses BigInt for the three fields and the second uses Base64.

Reviewer: ```suggestion
You can use BigInt or Base64 encoding for `extendedEphemeralPublicKey`, `jwtRandomness`, and `salt`. The following examples show two sample requests with the first using BigInt encoding and the second using Base64.
```
